PackageViewModel/SymbolValidation/SymbolValidator.cs

```diff
@@ -258,51 +258,64 @@ private async Task CalculateValidity(IReadOnlyList<PackageFile> files)
 
             var requireExternal = false;
             // See if any pdb's are missing and check for a snupkg on NuGet.org. 
-            if (noSymbols.Count > 0 && _publishedOnNuGetOrg)
+            if (noSymbols.Count > 0)
             {
-                // try to get on NuGet.org
-                // https://www.nuget.org/api/v2/symbolpackage/Newtonsoft.Json/12.0.3 -- Will redirect               
-
-                
                 try
                 {
+                    // try to find a sibling snupkg file locally
+                    var snupkgFilePath = Path.ChangeExtension(_packageViewModel.PackagePath, ".snupkg");
+                    if (File.Exists(snupkgFilePath))
+                    {
+                        ReadSnupkgFile(snupkgFilePath);
+                    }
+                    else if (_publishedOnNuGetOrg)
+                    {
+                        // try to get on NuGet.org
+                        // https://www.nuget.org/api/v2/symbolpackage/Newtonsoft.Json/12.0.3 -- Will redirect
+
 #pragma warning disable CA2234 // Pass system uri objects instead of strings
-                    var response = await _httpClient.GetAsync($"https://www.nuget.org/api/v2/symbolpackage/{_package.Id}/{_package.Version.ToNormalizedString()}").ConfigureAwait(false);
+                        var response = await _httpClient.GetAsync($"https://www.nuget.org/api/v2/symbolpackage/{_package.Id}/{_package.Version.ToNormalizedString()}").ConfigureAwait(false);
 #pragma warning restore CA2234 // Pass system uri objects instead of strings
 
-                    if (response.IsSuccessStatusCode) // we'll get a 404 if none
-                    {
-                        requireExternal = true;
+                        if (response.IsSuccessStatusCode) // we'll get a 404 if none
+                        {
+                            requireExternal = true;
```

Comment:
We need to set `requireExternal` to true for the local `snupkg` case as well. This is because it'll ultimately require a symbol server to work.